planet/order_request.py

```diff
@@ -22,6 +22,9 @@
 
 LOGGER = logging.getLogger(__name__)
 
+BAND_MATH_PIXEL_TYPE = ('Auto', '8U', '16U', '16S', '32R')
```

Comment:
this looks identical to https://github.com/planetlabs/planet-client-python/blob/main/planet/subscription_request.py#L21. I recommend moving this to specs.py and having both pull from there so we only have one place to update this if it ever changes.

Reply:
Yeah, great call!